src/index.ts

```diff
@@ -3,6 +3,7 @@ export { themes }
 
 export { default as Accordion } from './components/Accordion'
 export { default as Button } from './components/Button'
+export { ButtonGroup } from './components/Button'
 export { default as Chat } from './components/Chat'
 export { default as ChatMessage } from './components/Chat'
```

Comment:
seems that we have a problem here - as `Chat` is exported as `ChatMessage`. Could you, please, fix this? :)